apps/vast/config.yaml.example

```diff
@@ -10,3 +10,4 @@ unflatten: true
 transform_context: fever alertify --alert-prefix 'MY PREFIX' --extra-key my-ioc --ioc %ioc
 # optional. remove the field if you simply want to report back sightings to Threat Bus
 sink: STDOUT
+ulimit: 100
```

Comment:
Is this option documented elsewhere for the user?

Reply:
Yes, in the help menu when you invoke `threabus -h`.

Reply:
I added an additional comment.